src/styles/_nav.scss

```diff
@@ -0,0 +1,48 @@
+@import "variables";
```

Comment:
you don't need to import this in every file, you can import it one time in `main.scss`

Reply:
![image](https://github.com/mate-academy/layout_catalog/assets/54949928/60bf73d8-593a-414b-a7a9-2c035918e5c1)
Hi! Thank you for your review. It doesn't work.... =(